sdk/connectedvmwarevsphere/Azure.ResourceManager.ConnectedVMwarevSphere/tests/tests/Tests/ClusterTests.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.TestFramework;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.Core;
+using System.Linq;
+using Azure.ResourceManager.ConnectedVMwarevSphere.Models;
+using Azure.ResourceManager.ConnectedVMwarevSphere.Tests.Helpers;
+
+namespace Azure.ResourceManager.ConnectedVMwarevSphere.Tests.tests.Tests
+{
+    public class ClusterTests : ConnectedVMwareTestBase
+    {
+        public ClusterTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private async Task<VMwareClusterCollection> GetVMwareClusterCollectionAsync()
+        {
+            var resourceGroup = await CreateResourceGroupAsync();
+            return resourceGroup.GetVMwareClusters();
+        }
+
+        [AsyncOnly]
```

Comment:
Why do we add `AsyncOnly` for all test cases?

Reply:
By the way, we can extract invocation of `CreateOrUpdate` into a `Setup` method in the test cases, so that the creation is not duplicated in the test cases.

Reply:
Actually we can create connectedvmware rp resources with same "moref id" at only one time, in order to create an other resources with same moref id we need to delete the previous one. Since we are creating 5 resources for each resource type but we dont have 5 different moref id for the resources hence in order to create new resource deleting previous one is compulsory.